packer/openstack.pkr.hcl

```diff
@@ -142,10 +166,14 @@ build {
 # The "fat" image build with all binaries:
 build {
   source "source.openstack.openhpc" {
+    volume_size = "${var.volume_size}"
+    use_blockstorage_volume = "${var.use_blockstorage_volume}"
+    image_disk_format = "${var.image_disk_format}"
+    metadata = "${var.metadata}"
     floating_ip_network = "${var.floating_ip_network}"
```

Comment:
Should this be set in the compute build block too?

Reply:
I didn't think it made much sense to have a FIP attached to a compute node in the context of imaging a whole cluster, so I only added to the fatimage builder. I've not used the compute image builder, so don't really know if that is right or not - happy to update if needed, but will add it in a separate PR.

Reply:
The builder isn't really a compute node (just in that group, for this build), so if you need a FIP to access your builder you'll need a FIP here too. But I'm happy to skip that here as I think it's client-specific.